OracleIdentityGovernance/dockerfiles/12.2.1.4.0/oraInst.loc

```diff
@@ -0,0 +1,8 @@
+# # Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please remove copyright and license headers from all `.loc` files.

Reply:
done